sockeye/inference.py

```diff
@@ -862,17 +912,22 @@ def _get_inference_input(self,
         for j, trans_input in enumerate(trans_inputs):
             num_tokens = len(trans_input)  # includes eos
             max_output_lengths.append(self._get_max_output_length(num_tokens))
```

Comment:
I am not sure yet about whether we really want to exclude the prefix tokens from the maximum output length calculation (which is based on the length ratio estimated on the training data). The reason is that if your model can expect special tokens at the beginning, these tokens are have likely also been added to the training data, effectively impacting the source length statistics of the model. 
So maybe we should simply change line 913 to `num_tokens = len(trans_input) + trans_input.num_source_prefix_tokens()`?

Reply:
Yes I agreed we should take prefix tokens into account indeed.